lib/Swagger/src/ContentEntity/Api/RelatedContentApi.php

```diff
@@ -99,6 +99,128 @@ public function setApiClient(\Swagger\Client\ApiClient $apiClient)
     }
 
     /**
+     * Operation getOrderedRelatedContent
+     *
+     * get the content related to an entity
+     *
+     * @param string $wiki_id  (required)
+     * @param int $limit  (optional, default to 20)
+     * @param bool $include_root_relations  (optional, default to true)
+     * @param bool $show_connection_details  (optional, default to false)
+     * @param bool $restrict_universe  (optional, default to true)
+     * @param string[] $content_order  (optional)
+     * @return \Swagger\Client\ContentEntity\Models\MixedRelatedContent
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     */
+    public function getOrderedRelatedContent($wiki_id, $limit = null, $include_root_relations = null, $show_connection_details = null, $restrict_universe = null, $content_order = null)
+    {
+        list($response) = $this->getOrderedRelatedContentWithHttpInfo($wiki_id, $limit, $include_root_relations, $show_connection_details, $restrict_universe, $content_order);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$response". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)